src/table/settings.ini

```diff
@@ -3044,11 +3044,8 @@ strhelp  = STR_CONFIG_SETTING_WARN_LOST_VEHICLE_HELPTEXT
 
 [SDTC_BOOL]
 var      = gui.disable_unsuitable_building
-flags    = SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC
-def      = true
-str      = STR_CONFIG_SETTING_DISABLE_UNSUITABLE_BUILDING
-strhelp  = STR_CONFIG_SETTING_DISABLE_UNSUITABLE_BUILDING_HELPTEXT
-proc     = RedrawScreen
+flags    = SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC | SLF_NOT_IN_CONFIG
+def      = false
 cat      = SC_EXPERT
```

Comment:
I think the entire block needs to be deleted to fix the compile error. Since it's not in save and not network synced it shouldn't be a problem to remove it from the settings order.